commands/Moderation/kick.js

```diff
@@ -9,16 +9,20 @@ module.exports = class extends Command {
 			runIn: ['text'],
 
 			description: 'Kicks a mentioned user. Currently does not require reason (no mod-log).',
-			usage: '<member:member> [reason:string] [...]',
+			usage: '<member:user> [reason:string] [...]',
```

Comment:
the member argument already fetches the member, this should be left as member not user

Reply:
This is due to already established conventions in ban and softban